docs/pages/contributing.md

```diff
@@ -1,58 +1,96 @@
+    ``git checkout -b BRANCH-NAME-HERE``
 
-### Rebasing
+5. Make the appropriate changes for the issue you are trying to address or the feature you'd like to add.
 
-You should rarely develop or make changes to your local `main` branch, but sometimes, your `main` branch gets out of date. Also, some pull requests may have been merged that might require
-that the branch on which you are developing contains the latest commits.
+6. Add the file contents of the changed files to the "snapshot" git uses to manage the state of the project (also known as the index). Here's the git command that will add your changes:
 
-1. Make sure your `main` branch is up-to-date from the CrowdStrike repository before creating new branches. [Example on syncing a fork](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/syncing-a-fork)
-1. Once your `main` branch has been synced, rebase your working branch (before you create a pull request) by checking out your branch and then running `git rebase -i main`.
-2. For handling merge conflicts, see [how to resolve a git merge conflict](https://opensource.com/article/20/4/git-merge-conflict) for a walkthrough on handling merge conflicts.
+    ``git add insert-paths-of-changed-files-here``
 
-#### Squashing
+7. Use `git commit` to store the contents of the index with a descriptive message. This message should outline what was changed. For example:
 
-Before submitting pull requests, please make sure to squash commits in your branch when:
+    ``git commit -m "Added Dockerfile for Ubuntu-based deployments"``
 
-1. You create multiple commits (use `git log` to show history) that have the same commit message like `Update Readme.md`.
-1. Created a bunch of junk or throw away commit messages like `Fix`, `Test`, `Fixed Typos`, etc.
+8. Push your local changes back to gihub.com:
 
-[Example on squashing commits using git rebase](https://garrytrinder.github.io/2020/03/squashing-commits-using-interactive-rebase)
+    ``git push origin BRANCH-NAME-HERE``
+
+9. Submit a pull request to the upstream project. Documentation on this process, including screen shots, can be found here:
+
+[https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork)
+
+10. Once submitted, a maintainer will review your pull request. They may ask for additional changes, or clarification, so keep an eye out for communication! GitHub automatically sends an email to your email address whenever someone comments on your pull request.
+
+11. Celebrate your contribution after your pull request is merged! All changes move the project forward, and we thank you for helping the community!
```

Comment:
```suggestion
11. While not all pull requests may be merged, celebrate your contribution whether or not your pull request is merged! All changes move the project forward, and we thank you for helping the community!
```

I am not sure how to say this better, but I feel like we need to acknowledge in some way that some PRs might not get merged.